Feature/players add kardex

Adritay opened the pull request.

Adritay committed:
[feature] Increase fields to the player-model.
Description
========
Added fields to the player-model.
Added sub-documents to the player-model.

Adritay committed:
[feature]Verbs team-route and player-route file
Description
========
Added delete function to the team-route and player-route files.
Added update function to the team-route and player-route files.

david-suarez commented on models/player-model.js:
```diff
@@ -13,13 +47,33 @@ var PlayerSchema = new Schema({
         'default': "",
         'required': true
     },
+    'city': {
```
Please change the name of this variable to something more significant like city_of_birth.

Adritay committed:
[feature]Correcting code
Description
========
Changed name's fields from player-model file.

Adritay committed:
[feature]Correcting protocol's codes
Description
========
Changed the protocol codes of response in player-route and team-route files.
